Packages/com.unity.inputsystem/InputSystem/Events/InputEventPtr.cs

```diff
@@ -95,6 +95,12 @@ public int deviceId
         public double time
         {
             get { return valid ? m_EventPtr->time : 0.0; }
+            set
+            {
+                if (!valid)
+                    throw new NullReferenceException();
```

Comment:
Bad precedent set by me. `NullReferenceException` shouldn't be thrown by user code.